forge/README.md

```diff
@@ -131,11 +131,15 @@ which implements the following methods:
 
 - `function deal(address who, uint256 amount)`: Sets an account's balance
 
-- `function etch(address where, bytes memory what)`:` Sets the contract code at
+- `function etch(address where, bytes memory what)`: Sets the contract code at
   some address contract code
 
-- `function prank(address from, address to, bytes calldata) (bool success,bytes retdata)`:
-  Performs a smart contract call as another address
+- `function prank(address from)`: Performs the next smart contract call as another address
+
+- `function prankStart(address from)`: Performs smart contract calls as another address
```

Comment:
they are called startPrank and stopPrank, if you can change that

Reply:
Fixed it, got confused with the PRs old names